app/controllers/pdf_exports_controller.rb

```diff
@@ -65,13 +65,18 @@ def text_blocks(certificate, certificate_template)
         text_box_options: {
           character_spacing: entry.character_spacing,
           size: entry.font_size,
+          color: format_color(entry.color),
           at: [entry.x, entry.y],
           align: entry.align.to_sym
         }
       }
     end
   end
 
+  def format_color(color)
+    color.start_with?('#') ? color.slice(1, color.length - 1) : color
```

Comment:
nit: Or we could strip the `#` sign before saving the color to the DB. Optimization geeks would say it saves a lot of DB storage in a long run :nerd_face:

Reply:
Storing colors with # is consistent with the HTML/CSS standard and simplifies client-side rendering by eliminating the need for additional processing. It also reduces errors and makes data usage more straightforward, and the database size is virtually unchanged.